config.m4

```diff
@@ -168,9 +168,22 @@ if test "$PHP_REDIS" != "no"; then
     MSGPACK_INCLUDES=""
   fi
 
+  AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+
   if test "$PHP_REDIS_LZF" != "no"; then
     AC_DEFINE(HAVE_REDIS_LZF, 1, [ ])
-    if test "$PHP_LIBLZF" != "no"; then
+
+    if test "$PHP_LIBLZF" = "yes" && test -x "$PKG_CONFIG" && $PKG_CONFIG --exists liblzf; then
```

Comment:
`./configure --with-liblzf=yes` should be used to enable pkg-config, right?

Reply:
yes, or without option

Reply:
in short:
* `--enable-redis-lzf` => use system library from pkg-config
* `--enable-redis-lzf `--with-liblzf=/foo/bar => use alternative library (deprecated as since 7.4 mostly everything use `pkg-config` "only" in php-src)